packages/batch-submitter/test/batch-submitter/batch-submitter.spec.ts

```diff
@@ -484,7 +483,7 @@ describe('Batch Submitter with Ganache', () => {
       gasPrices.push(gasPrice)
 
       const tx = signer.sendTransaction({
-        to: DECOMPRESSION_ADDRESS,
+        to: predeploys.Lib_AddressManager,
```

Comment:
Cc @karlfloersch @annieke or whoever wrote these tests. 0x42....08 is definitely the address manager predeploy but the constant here suggests you meant to use the sequencer entrypoint. Can someone shine light here as to which it should be?

Reply:
Should also be updated to `predeploys.OVM_SequencerEntrypoint`.